replay_build_scripts/build-pipeline.py

```diff
@@ -25,7 +25,10 @@ def generate_buildkite_pipeline(backend_commit_hash):
             {
                 "trigger": "chromium-build",
                 "build": {
-                    "env": {"DRIVER_REVISION": driver_revision},
+                    "env": {
+                        "DRIVER_REVISION": driver_revision,
+                        "REPLAY_BACKEND_REV": backend_commit_hash,
```

Comment:
Should the 2 values not always be the same?

Reply:
The only difference is that `driver_revision` is truncated to 12 characters, whereas `backend_commit_hash` is the entire hash.